lib/wsdl.js

```diff
@@ -2081,14 +2082,20 @@ WSDL.prototype._fromServices = function(services) {
 
 WSDL.prototype._splitQName = splitQName;
 
-WSDL.prototype._xmlnsMap = function() {
+WSDL.prototype._xmlnsMap = function (xmlnsIntoEnvelope) {
   var xmlns = this.definitions.xmlns;
   var str = '';
   for (var alias in xmlns) {
     if (alias === '' || alias === TNS_PREFIX) {
       continue;
     }
+
     var ns = xmlns[alias];
+    if (ns && xmlnsIntoEnvelope && ~xmlnsIntoEnvelope.indexOf(alias)) {
```

Comment:
But, if xmlnsIntoEnvelope does not exists, will _.includes throw an Error? i.e. xmlnsIntoEnvelope not defined. Of course, I think your solution is better than mine.